pkgs/applications/editors/vscode/extensions/_maintainers/update-bin-srcs-lib.sh

```diff
@@ -37,14 +37,12 @@ prefetchExtensionUnpacked() {
   1>&2 echo "zipStorePath='$zipStorePath'"
 
   function rm_tmpdir() {
-    1>&2 printf "rm -rf -- %q\n" "$tmpDir"
-    rm -rf -- "$tmpDir"
-    unset tmpDir
-    trap - INT TERM HUP EXIT
+    1>&2 printf "rm -rf %q\n" "$tmpDir"
+    rm -rf "$tmpDir"
```

Comment:
Do you not want to preserve the end-of-options `--` here?

Reply:
No, we also don't do this at other places in nixpkgs. `tmpDir` is guaranteed to never begin with a `-`.

Reply:
I've extracted this change to a separate commit.